resources/public/api/conf/1.0/schemas/lossClaims/listClaims/list_response.json

```diff
@@ -16,6 +16,11 @@
             "pattern": "^[A-Za-z0-9]{15}$",
             "example": "A2Za0911235813z"
           },
+          "sequence":  {
+            "description": "A number detailing the order in which a loss will be used, the value must be between 1 and 99",
```

Comment:
Can we double check the content for the "between X and Y" bit? There isn't anything in the spec for this text so I think we're best off reviewing it with the Content Designer.